internal/telemetry/default.go

```diff
@@ -173,14 +118,19 @@ func (t *defaultTelemetry) sendRecordedTelemetry() {
 		success := tags["success"] == "y"
 		summary := bucket.Summarize()
 
-		t.SendEvent(t.ctx, &OperationsPerformed{
-			Operation:       op,
-			Success:         success,
-			DurationSummary: summary,
-		})
+		t.SendEvent(t.ctx, OperationsPerformedEvent(op, success, summary))
 	}
 }
 
+func (t *defaultTelemetry) nextEventID() int {
```

Comment:
so event ids are scoped by session id and start at zero?

Reply:
Yes - event id is only used to deduplicate events from a single user that may have the same timestamp.